src/DB.php

```diff
@@ -804,14 +804,19 @@ public static function insertMessageRequest(Message $message)
 
         //New and left chat member
         if (!empty($new_chat_members)) {
+            $new_chat_members_ids = '';
```

Comment:
Reminding you of using an array for this and then `implode`ing:
```php
$ids = [];
$ids[] = 123;
$ids[] = 456;
$ids_csv = implode(',', $ids);
```

If you want to keep it using a string, what you can change is:
```php
// this...
if (!empty($new_chat_members_ids)) {

// ...to this
if ($new_chat_members_ids) {
```

As it's the same thing.